### DISCUSSION TITLE
level=info msg="failed to verify token: token signed by untrusted key with ID: \"AO3Q:SPP2:WIFU:75M6:KN7Y:3EOJ:AHJD:WCZD:YXSG:4GV7:4KIY:TKUI\""

### DISCUSSION THREAD

**Discussion by GuoYingLong** [distribution/distribution#4364] [Q&A]
In the process of docking, we encountered problems. Can anyone help me solve it?
    error message: level=info msg="failed to verify token: token signed by untrusted key with ID: \"AO3Q:SPP2:WIFU:75M6:KN7Y:3EOJ:AHJD:WCZD:YXSG:4GV7:4KIY:TKUI\""

**Comment by milosgajdos** [comment]
Related to this issue: https://github.com/distribution/distribution/issues/4299

You can take this PR for a spin: https://github.com/distribution/distribution/pull/4349
